addons/binding/org.openhab.binding.nest/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                          xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0" xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
```

Comment:
Is this the preferred way to channels btw?  Have them pulled out independent of the thing definition?

Reply:
Yes, because they are all in a shared namespace so duplicates will cause conflicts with each other.